lib/gollum/templates/navbar.mustache

```diff
@@ -0,0 +1,43 @@
+<nav class="actions pt-4">
+	
+	<div class="TableObject">
+		<div class="TableObject-item">
+	    <a class="btn" href="{{base_url}}/">Home</a>
+	  </div>
+
+
+  	<div class="TableObject-item TableObject-item--primary px-2" {{^search}}style="visibility:hidden"{{/search}}>
```

Comment:
Perhaps just `hidden` would be enough, to avoid inline CSS?

Reply:
Those do not appear to be equivalents, or at least it gives different results. I think someone with more know-how could tackle this one.